sync_jira_actions/sync_to_jira.py

```diff
@@ -106,14 +102,20 @@ def main():
         if 'pull_request' not in event['issue']:
             event['issue']['pull_request'] = True  # we don't care about the value
 
-    # don't sync if user is our collaborator
+    # Don't sync a PR if user/creator is a collaborator
     github = Github(os.environ['GITHUB_TOKEN'])
     repo = github.get_repo(os.environ['GITHUB_REPOSITORY'])
     gh_issue = event['issue']
     is_pr = 'pull_request' in gh_issue
     if is_pr and repo.has_in_collaborators(gh_issue['user']['login']):
         print('Skipping issue sync for Pull Request from collaborator')
         return
```

Comment:
Isn't this going to be an issue as we are all contributors?

Reply:
This was so we don't end up with duplicate jira issues for the same task. However, I just added a feature that allows collaborators to sync PR's when the specific sync label is added. 😄